src/nimib/highlight.nim

```diff
@@ -0,0 +1,94 @@
+import std/strutils
+from std/cgi import xmlEncode
+
+import packages/docutils/highlite
+
+func nimNormalize(s: string): string =
```

Comment:
what's the difference between this and https://nim-lang.org/docs/strutils.html#normalize%2Cstring ?

Reply:
@pietroppeter as explained in the comments, my modification actually compares nim identifiers, while that proc says "This should NOT be used to normalize Nim identifier names."

The difference is when we are comparing the first character of the string, it's case sensitive in Nim

Reply:
ok, I see it now, the difference with the strutils version is the `i == 0` branch that keeps case of first character (something that we need in fact).